circt.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
We have script folder in the jlm repository. I have the feeling this script would be better suited there.

Reply:
Moved

Reply:
I found out that we have a scipts folder, so renamed it to scripts.